### PR TITLE
Export selected region as a iiif url

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: src/iiifRegion.js [hunk 1]
```diff
@@ -47,7 +45,17 @@ IiifRegion.prototype = {
   y: function() {}, // getter/setter
   width: function() {}, // getter/setter
   height: function() {},
-  getRegion: function() {}
+  getRegion: function() {
```
Comment (jcoyne) [inline src/iiifRegion.js]: This should return an array (x, y, width, height)

File: src/iiifRegion.js [hunk 2]
```diff
@@ -47,7 +45,17 @@ IiifRegion.prototype = {
...
+    return this.x + ',' + this.y + ',' + this.width + ',' + this.height;
+  },
+
+  getUrl: function() {
```
Comment (jcoyne) [inline src/iiifRegion.js]: Move this to a service class

**Comment by aeschylus** [comment]
Nice. Thanks for also setting up travis.

**Comment by aeschylus** [comment]
Though there seem to be some problems with the tests.

**Comment by aeschylus** [comment]
Looks like the osd dependency is either registered incorrectly or is otherwise failing.

**Comment by jcoyne** [comment]
@aeschylus The tests are now passing.

**Comment by aeschylus** [comment]
Nice. Thanks Justin.

**Pull request merged**